charts/akeyless-secure-remote-access/values.yaml

```diff
@@ -367,15 +370,19 @@ sshConfig:
     fsGroup: 0
     runAsUser: 0
 
-  env: []
+  env:
+    # This env variable instruct docker to start only ssh-proxy-bastion processes
+    - name: "BASTION"
```

Comment:
same on the name

Reply:
dont add this to env part.. put it in a separate section

Reply:
done